installer/dev/install.cpp

```diff
@@ -191,9 +189,22 @@ namespace WindowsAppRuntimeInstaller
             return true;
         }
 
-        // On Arm64 systems, all current package architectures are applicable.
+        // On Windows 11 (i.e. builds 22000+) ARM64 systems, all framework package architectures are applicable.
+        // On Windows 10 (i.e. builds 17763-190**) ARM64 systems (which don't support X64 apps), all but x64 framework package architecture is not applicable.
```

Comment:
`all but x64 framework package architecture is not applicable`

Odd phrasing, had to re-read a few times to be sure I understood it

SUGGESTION: `..., all framework package architectures are applicable except x64.`

That reads like the Win11 line above "...except x64"